http/http-minimum-reactive/src/test/java/io/quarkus/ts/http/minimum/reactive/HttpMinimumReactiveIT.java

```diff
@@ -78,7 +86,27 @@ public void shortRecordReturnedDirectly() {
                 .body("data", is("ok"));
     }
 
+    @Test
+    void executePostOperator() {
```

Comment:
And maybe I'd call it differently. I don't want to be too annoying as we are close to Christmas, but probably not a best name? I'll leave you decide as I have already approved.

Reply:
This is the one used in author's reproducer. I do not mind changing it to any other, since I have nothing else to do before our Christmas' time off (this is not a sarcasm). Could you choose more suitable name?

Reply:
Name changed (I took the liberty to change suggested name to be more in style with the rest of the class), tag added.